_posts/2023-05-31-socratic-seminar-124.md

```diff
@@ -107,62 +86,11 @@ If you can't make it to the main event please join us at PUBKEY around 9:30PM. *
 ## Pull Requests and repo updates
 ### [Bitcoin Core](https://github.com/bitcoin/bitcoin)
```

Comment:
It might not have been in the 100 PRs retrieved from the github API:

https://api.github.com/repos/bitcoin/bitcoin/pulls?state=closed&sort=closed&direction=desc&per_page=100&page=1

https://github.com/pinheadmz/bitdevs-scraper/blob/5de15fe43e2f81b31bb75ce405faa0f1bee183c0/scrape.js#LL188C16-L188C125

will tune up for next month!